src/nanorc/logbook.py

```diff
@@ -119,34 +119,35 @@ def _send_message(self, subject:str, body:str, command:str):
 
 
 
-    def message_on_start(self, message:str, apparatus:str, run_num:int, run_type:str):
+    def message_on_start(self, messages:[str], session:str, run_num:int, run_type:str):
         self._start_new_message_thread()
         self.current_run_num = run_num
         self.current_run_type = run_type
 
-        user = self.session_handler.nanorc_user.username
 
-        text = f"<p>User {user} started run {self.current_run_num} of type {self.current_run_type} on {apparatus}</p>"
-        if message != "":
-            text += f"\n<p>{user}: {message}</p>"
+        text = ''
+        for message in messages:
+            text += f"\n<p>{message}</p>"
         text += "\n<p>log automatically generated by NanoRC.</p>"
 
-        title = f"{user} started new run {self.current_run_num} ({self.current_run_type}) on {apparatus}"
+        title = f"Run {self.current_run_num} ({self.current_run_type}) started on {session}"
         self._send_message(subject=title, body=text, command='start')
 
-    def add_message(self, message:str, apparatus:str):
-        user = self.session_handler.nanorc_user.username
 
-        if message != "":
-            text = f"<p>{user}: {message}</p>"
+    def add_message(self, messages:[str], session:str):
+
+        for message in messages:
+            text = f"<p>{message}</p>"
             self._send_message(subject="User comment", body=text, command='message')
 
-    def message_on_stop(self, message:str, apparatus:str):
-        user = self.session_handler.nanorc_user.username
-        text = f"<p>User {user} finished run {self.current_run_num} on {apparatus}</p>"
-        if message!="":
-            text = f"\n<p>{user}: {message}</p>"
+
+    def message_on_stop(self, messages:[str], session:str):
+        text = ''
+
+        for message in messages:
+            text = f"\n<p>{message}</p>"
+
         text += "\n<p>log automatically generated by NanoRC.</p>"
 
-        title = f"{user} ended run {self.current_run_num} ({self.current_run_type}) on {apparatus}"
+        title = f"Run {self.current_run_num} ({self.current_run_type}) stopped on {session}"
```

Comment:
Not seeing this title in the logbook. Instead, just seeing a 'reply' message to the start, and nothing to indicate that it stopped. (see https://pdsp-elog.app.cern.ch/elisa/display/20268?logbook=ProtoDUNE-SP)

Reply:
Indeed, titles cannot be changed in responses (which I guess makes sense).

Oh well, it'll have to be in the body of the message then.